index.html

```diff
@@ -1,12 +1,336 @@
+                </p>
+                <div class="row py-2">
+                  <div class="col-6">
+                    <span class="h5 fw-bold text-center text-white"
+                      >₦ 9,000</span
+                    >
+                  </div>
+                  <div class="col-6 text-end">
+                    <span class="p-2 bg-light rounded-3">
+                      <img
+                        src="Assets/cart-vector.png"
+                        alt=""
+                        class="img-fluid"
+                      />
+                    </span>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-12 col-md-6 col-lg-3 mt-5 mt-lg-0">
+            <div class="card border-0 card-bg-4">
+              <div class="card-body">
+                <h5 class="card-title text-white text-center mt-5 pt-5">
+                  Italian Pasta
+                </h5>
+                <p class="card-text text-white text-center">12,456 &#11088;</p>
+                <p class="text-uppercase text-white">
+                  <small>Mashed potatoes with T-bone steak...</small>
+                </p>
+                <div class="row py-2">
+                  <div class="col-6">
+                    <span class="h5 fw-bold text-white">₦ 9,000</span>
+                  </div>
+                  <div class="col-6 text-end">
+                    <span class="p-2 bg-light rounded-3">
+                      <img
+                        src="./images/cart-vector.png"
+                        alt=""
+                        class="img-fluid"
+                      />
+                    </span>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section id="dishes" class="py-5">
+      <div class="dishes-wrapper">
+        <div class="container1">
+          <div class="row">
+            <div class="col-12">
+              <div class="text-center centered-title mb-5 pb-lg-4">
+                <h2 class="text-black">Some Dishes</h2>
+                <div class="line"></div>
+              </div>
+            </div>
+          </div>
+          <div id="get-dishes" class="row"></div>
+          <div class="row mt-3 d-md-none d-lg-none d-xl-none">
+            <div class="col-12">
+              <div class="call-to-action pt-lg-5 pt-xl-5">
+                <a
+                  id="show-more-dishes"
+                  class="show-more-dishes btn btn-outline-dark w-100 d-flex align-items-baseline justify-content-center"
+                  href="#"
+                >
+                  <div class="me-2">More</div>
+                  <i class="fa fa-angle-down orange-text"></i>
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
Kindly fix your spacing, font, and color to match the required design
Your Design
![image](https://user-images.githubusercontent.com/66112760/170843628-e5789482-d8c5-465f-a66b-f832ecea8deb.png)

Figma Design
![image](https://user-images.githubusercontent.com/66112760/170843640-39a2ccbf-b2ab-4381-b6d0-aedb9847a54a.png)